lua/rip-substitute/popup-win.lua

```diff
@@ -67,11 +68,17 @@ end
 
 local function confirmSubstitution()
```

Comment:
I think for clarity reasons, this should now be renamed to `confirmSingleSubstitution`

Reply:
makes sense 👍 renamed it